docs/custom-slashing/dd.template.md

```diff
@@ -25,48 +39,99 @@ The requirements with filled checkboxes are met by the design below.
   - How/where to vote/signal on L1
 - [x] Node operators SHOULD be able to configure their node to specify thresholds for what they consider "not participating".
 - [x] The "offence" that triggers the slash MAY be specified on L1.
-- [ ] The amount to be slashed MAY be configurable without deploying a new contract.
+- [x] The amount to be slashed MAY be configurable without deploying a new factory contract.
 - [ ] The threshold of number of validators (M/N) that need to signal/vote for the CustomSlashFactory payload MAY be configurable without deploying a new contract or a governance action.
 
-## Overview
+## L1 Changes
+
+We make no changes to the `Slasher` contract, or any other "in-protocol" contracts.
+
+Refactor the `SlashFactory` to accept an array of validator addresses, amounts, and offences. I.e.
+
+```solidity
+interface ISlashFactory {
+  enum Offense {
```

Comment:
I would **NOT** use an enum, as it is then specific to what we conjured up now, I would just use a number, such that it can be handled "off-chain" to map it to something specific.